src/docs/stan-reference/distributions.tex

```diff
@@ -1911,13 +1911,12 @@ \subsubsection{Probability Density Function}
 \distro{Wiener}(y|\alpha, \tau, \beta, \delta)
 =
 \frac{\alpha}{(y-\tau)^3/2}
-\,
+%
```

Comment:
what's with the changes to the distributions? I haven't seen it built yet, but I wanted to make sure these were intentional.